### PR TITLE
ROC-5334 route test improvements

### PR DESCRIPTION
**Pull request by robertparkinson**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-5334

### Change description

Add in route tests to help increase resilience and move some tests to nightly

### Work checklist

- [x] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/test/features/claim/routes/interest-rate.ts [hunk 1]
```diff
@@ -56,6 +56,32 @@ describe('Claim issue: interest rate page', () => {
...
+          .post(pagePath)
+          .set('Cookie', `${cookieName}=ABC`)
+          .send({
+            type: InterestRateOption.DIFFERENT,
+            reason: 'Special case'
+          })
+          .expect(res => expect(res).to.be.successful.withText(pageContent, 'div class="error-summary"'))
+      })
+
+      it('should render the page when form when a different rate is selected but no reason is entered', async () => {
```
Comment (andrewwa-kainos) [inline src/test/features/claim/routes/interest-rate.ts]: These two read strangely - remove "when form" from both?

File: src/test/features/claim/routes/total.ts [hunk 2]
```diff
@@ -7,13 +7,14 @@ import 'test/routes/expectations'
...
 
-import { Paths as ClaimPaths } from 'claim/paths'
+import { ErrorPaths, Paths as ClaimPaths } from 'claim/paths'
 
 import { app } from 'main/app'
 
 import * as idamServiceMock from 'test/http-mocks/idam'
 import * as draftStoreServiceMock from 'test/http-mocks/draft-store'
 import * as feesServiceMock from 'test/http-mocks/fees'
+import * as claimStoreServiceMock from '../../../http-mocks/claim-store'
```
Comment (andrewwa-kainos) [inline src/test/features/claim/routes/total.ts]: Can we keep these non-relative?

File: src/test/features/claim/routes/total.ts [hunk 3]
```diff
@@ -89,6 +90,17 @@ describe('Claim issue: total page', () => {
...
+
+      it('should throw error when claim value is above £10000 including interest', async () => {
+        draftStoreServiceMock.resolveFind('claim', draftStoreServiceMock.aboveAllowedAmountWithInterest)
+        claimStoreServiceMock.mockCalculateInterestRate(0)
+        claimStoreServiceMock.mockCalculateInterestRate(500)
+
+        await request(app)
+          .get(pagePath)
+          .set('Cookie', `${cookieName}=ABC`)
+          .expect(res => expect(res).to.be.redirect.toLocation(ErrorPaths.amountExceededPage.uri))
```
Comment (andrewwa-kainos) [inline src/test/features/claim/routes/total.ts]: Note this clashes with https://github.com/hmcts/cmc-citizen-frontend/pull/1469

File: src/test/features/claim/routes/interest-rate.ts (hunk 1 above)
Comment (robertparkinson) [reply src/test/features/claim/routes/interest-rate.ts]: done

File: src/test/features/claim/routes/total.ts (hunk 2 above)
Comment (robertparkinson) [reply src/test/features/claim/routes/total.ts]: done

File: src/test/features/claim/routes/total.ts (hunk 3 above)
Comment (robertparkinson) [reply src/test/features/claim/routes/total.ts]: yeah I'm expecting to have to change this when that gets merged but my change may get in before that so left it with the currently working fix.

**Pull request merged**